tests/test_mars_format.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from unittest.case import TestCase
+import os.path as osp
+
+import numpy as np
+
+from datumaro.components.annotation import Label
+from datumaro.components.dataset import Dataset, DatasetItem
+from datumaro.components.environment import Environment
+from datumaro.plugins.mars_format import MarsImporter
+from datumaro.util.test_utils import compare_datasets
+
+from tests.requirements import Requirements, mark_requirement
+
+ASSETS_DIR = osp.join(osp.dirname(__file__), 'assets')
+DUMMY_MARS_DATASET = osp.join(ASSETS_DIR, 'mars_dataset')
+
+class MarsImporterTest(TestCase):
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_import(self):
+        expected_dataset = Dataset.from_iterable([
+            DatasetItem(id='0001C1T0001F001', image=np.ones((10, 10, 3)),
+                subset='train', annotations=[Label(label=2, attributes={
+                    'is_distractors': False,
+                    'is_junk': False,
+                    'pedestrian_id': '0001',
+                    'camera_id': '1',
+                    'track_id': '0001',
+                    'frame_id': '001'})]
+            ),
+            DatasetItem(id='0000C6T0101F001', image=np.ones((10, 10, 3)),
+                subset='train', annotations=[Label(label=1, attributes={
+                    'is_distractors': True,
+                    'is_junk': False,
+                    'pedestrian_id': '0000',
+                    'camera_id': '6',
+                    'track_id': '0101',
+                    'frame_id': '001'})]
+            ),
+            DatasetItem(id='00-1C2T0081F201', image=np.ones((10, 10, 3)),
+                subset='test', annotations=[Label(label=0, attributes={
+                    'is_distractors': False,
+                    'is_junk': True,
+                    'pedestrian_id': '00-1',
+                    'camera_id': '2',
+                    'track_id': '0081',
+                    'frame_id': '201'})]
+            ),
+        ], categories=['00-1', '0000', '0001'])
```

Comment:
The LFW, WiderFace and MARS are related to the same task, consider aligning their representations.

Reply:
What do you mean by "aligning"? Do you mean that the formats should have a similar representation in Datumaro or something else?

Reply:
Yes, exactly. It will look pretty hard to use if these formats are not convertible between each other.